src/amuse/test/suite/core_tests/test_store.py

```diff
@@ -28,7 +29,8 @@ def store_version(self):
         
     def get_version_in_store(self, container):
         raise NotImplementedError
-        
+
+    @pytest.mark.skip
```

Comment:
don't think this should be the solution...

Reply:
no

Reply:
how about adding an is_abstract method to the class, skip if it returns True?

Reply:
seems clumsy but I can't figure out a better method...

Reply:
never mind, can just add @unittest.skip() to a class

Reply:
but then that gets inherited... brrrr....